### DISCUSSION TITLE
Add the app version in the "Help & Feedback" menu

### DISCUSSION THREAD

**Discussion by Parkertg** [standardnotes/forum#2677] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?** -Yup

**Describe the solution you'd like**
I'd like for the app version to be listed in the Help & Feedback menu

**Is your feature request related to a problem? Please describe.**
I keep opening the "Help & feedback" preferences menu to find the APP version. I think its fairly clear that the team agrees that "Help & Feedback" is the right place as the only place I've found it is on a button linking to that menu (shown in image below).
I don't think its the intuitive to click on what I see as the "Account Icon" to get the app's version.. not that I mind it there. Just seems it should be in the menu that it's linking to as well

**Additional context**
this is the only place i can find it in the webapp
![image](https://user-images.githubusercontent.com/46120246/196587627-092c2ba2-e3a4-4446-9d26-d89e4a8cb88f.png)

**Comment by myreli** [comment]
Great suggestion. Alternatively, we can see the current version as well as available updates under "What's New". 

<img width="894" alt="image" src="https://user-images.githubusercontent.com/17554234/218866322-2d784c68-c770-44c1-ba72-3f8ec73ad20b.png">
